### DISCUSSION TITLE
SSO with 2 applications : one in SAML another in OpenID

### DISCUSSION THREAD

**Discussion by ririsoft** [keycloak/keycloak#11870] [Q&A]
Hello,

Does Keycloak support SSO with 2 applications that use different protocols : one is using SAML another is using OpenID, please ?
Thanks in advance for your help.
Cheers.

**Comment by bjolivot** [comment]
Yes

**Reply by ririsoft** [reply]
Looking at https://stackoverflow.com/questions/65018357/keycloak-sharing-session-across-protocols-saml-oidc
it seems not possible to share sessions between SAML & OIDC client applications, you need one session for each.

Do you confirm that on your side we can have 1 shared KC session for 2 applications using different protocols ?

**Reply by darius-m** [reply]
Could you please describe your use case further? Why would having a shared session between applications be a requirement?

**Reply by Aleson95** [reply]
please elaborate more on how we can achieve this
